fdbclient/admin_client.go

```diff
@@ -49,7 +49,7 @@ type timeoutError struct {
 }
 
 func (timeoutErr timeoutError) Error() string {
-	return fmt.Sprintf("timeout: %s", timeoutErr.err.Error())
+	return fmt.Sprintf("fdbcli timeout: %s", timeoutErr.err.Error())
```

Comment:
Looking at this again, I think we should move the timeout error into the API and do an actual type check instead of checking a string.